specification/web/resource-manager/Microsoft.Web/stable/2016-03-01/ResourceHealthMetadata.json

```diff
@@ -0,0 +1,373 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceHealthMetadataCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "List ResourceHealthMetadata for a site": {
+            "$ref": "./examples/ListResourceHealthMetadataBySite.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Web/sites/{name}/slots/{slot}/resourceHealthMetadata/default": {
+      "get": {
+        "tags": [
+          "ResourceHealthMetadata"
+        ],
+        "summary": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "description": "Gets the category of ResourceHealthMetadata to use for the given site",
+        "operationId": "ResourceHealthMetadata_GetBySiteSlot",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "name",
+            "in": "path",
+            "description": "Name of web app",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "slot",
+            "in": "path",
+            "description": "Name of web app slot. If not specified then will default to production slot.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ResourceHealthMetadataCollection"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get ResourceHealthMetadata": {
+            "$ref": "./examples/GetResourceHealthMetadataBySite.json"
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": "nextLink"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "ResourceHealthMetadata": {
+      "description": "Used for getting ResourceHealthCheck settings.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "ResourceHealthMetadata resource specific properties",
+          "properties": {
+            "category": {
```

Comment:
Is this a candidate for an enum?

Reply:
This is a string as per spec provided by the RHC team, relevant part of spec:
Category (string) - this is the category of the sku that you have communicated to RHC. RHC Rp will select the policy based on this value. If the sku's category does not match existing categories, we will fall back to the default category